release/apparmor.go

```diff
@@ -184,10 +184,25 @@ func MockAppArmorFeatures(kernelFeatures, parserFeatures []string) (restore func
 // probe related code
 
 var (
+	// requiredAppArmorParserFeatures denotes the features that must be present in the parser.
+	// Absence of any of those features results in the effective level be at most UnusableAppArmor.
 	requiredAppArmorParserFeatures = []string{
 		"unsafe",
 	}
+	// preferredAppArmorParserFeatures denotes the features that should be present in the parser.
+	// Absence of any of those features results in the effective level be at most PartialAppArmor.
+	preferredAppArmorParserFeatures = []string{
+		"unsafe",
```

Comment:
It doesn't make sense (to me anyway) to have something in preferred also in required. Shouldn't this just be empty?

Reply:
From one point of view you could say that what we require is a subset of what we prefer and not a distinct set. From another you can say that this can be empty since we already require that anyway. I don't think either is more correct than the other. In the implementation I opted into the first of those interpretations.